Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -287,6 +287,32 @@ PhysicalParticleContainer::PhysicalParticleContainer (AmrCore* amr_core, int isp
     }
 #endif
 
+    // User-defined integer attributes
```

Comment:
Looking at the attributes above: default attribute names and what we add for ionization/qed, we should add them to the doc section here: https://warpx.readthedocs.io/en/latest/developers/particles.html#runtime-particle-attributes

Reply:
I am not sure it should go there?
That lists implementation details.

So I added them in the doc where user input parameters for particle initialization are listed.
https://warpx.readthedocs.io/en/latest/usage/parameters.html#particle-initialization

Reply:
I'll push a commit to complete the list :)

Reply:
done